File: Bars.Gkh1468/libs/B4/model/publicservorg/contractpart/IndividualOwnerContract.js
Ext.define('B4.model.publicservorg.contractpart.IndividualOwnerContract', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'IndividualOwnerContract',
        writer: {
            type: 'b4writer',
            writeAllFields: true
        }
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'PublicServiceOrgContract', defaultValue: null },
        { name: 'TypeContractPart', defaultValue: null },
        { name: 'TypeContactPerson', defaultValue: null },
        { name: 'TypeOwnerContract', defaultValue: null },
        { name: 'FirstName', defaultValue: null },
        { name: 'LastName', defaultValue: null },
        { name: 'MiddleName', defaultValue: null },
        { name: 'Gender', defaultValue: null },
        { name: 'OwnerDocumentType', defaultValue: null },
        { name: 'IssueDate', defaultValue: null },
        { name: 'DocumentSeries', defaultValue: null },
        { name: 'DocumentNumber', defaultValue: null },
        { name: 'BirthPlace', defaultValue: null },
        { name: 'BirthDate', defaultValue: null }
    ]
});

File: Bars.Gkh/Modules/Gkh1468/Entities/PublicServiceOrg/ContractPart/IndividualOwnerContract.cs
namespace Bars.Gkh.Modules.Gkh1468.Entities.ContractPart
{
    using System;

    using Bars.Gkh.Modules.Gkh1468.Enums;

    /// <summary>
    /// Сторона договора "Физическое лицо"
    /// </summary>
    public class IndividualOwnerContract : BaseContractPart
    {
        /// <summary>
        /// Лицо, являющееся стороной договора
        /// </summary>
        public virtual TypeContactPerson TypeContactPerson { get; set; }

        /// <summary>
        /// Фамилия
        /// </summary>
        public virtual string LastName { get; set; }

        /// <summary>
        /// Имя
        /// </summary>
        public virtual string FirstName { get; set; }

        /// <summary>
        /// Отчество
        /// </summary>
        public virtual string MiddleName { get; set; }

        /// <summary>
        /// Пол
        /// </summary>
        public virtual GenderR Gender { get; set; }

        /// <summary>
        /// Тип документа
        /// </summary>
        public virtual OwnerDocumentType OwnerDocumentType { get; set; }

        /// <summary>
        /// Дата выдачи
        /// </summary>
        public virtual DateTime? IssueDate { get; set; }

        /// <summary>
        /// Серия документа
        /// </summary>
        public virtual string DocumentSeries { get; set; }

        /// <summary>
        /// Номер документа
        /// </summary>
        public virtual string DocumentNumber { get; set; }

        /// <summary>
        /// Место рождения
        /// </summary>
        public virtual string BirthPlace { get; set; }

        /// <summary>
        /// Дата рождения
        /// </summary>
        public virtual DateTime BirthDate { get; set; }
    }
}
